rusoto/signature/Cargo.toml

```diff
@@ -20,15 +20,25 @@ edition = "2018"
 appveyor = { repository = "matthewkmayer/rusoto", branch = "master" }
 azure-devops = { project = "matthewkmayer/Rusoto", pipeline = "rusoto.rusoto", build="1" }
 
+[features]
+default = ["futures-01"]
```

Comment:
Thanks for kicking the tires on the future features. 

I'm wondering if this could t have been a separate pull. In either case it would be worth it to make sure ci is covering the new feature flag combos.